codec.go

```diff
@@ -1,35 +1,63 @@
 package pnet
 
 import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"encoding/hex"
 	"fmt"
 	"io"
-
-	mc "github.com/multiformats/go-multicodec"
-	bmux "github.com/multiformats/go-multicodec/base/mux"
 )
 
 var (
-	pathPSKv1   = []byte("/key/swarm/psk/1.0.0/")
-	headerPSKv1 = mc.Header(pathPSKv1)
+	pathPSKv1  = []byte("/key/swarm/psk/1.0.0/")
+	pathBase16 = "/base16/"
+	pathBase64 = "/base64/"
```

Comment:
Multicodec also supported `bin`. IDK how widely it was deployed but it shouldn't be a problem.

Reply:
I mean, I can just support `/bin/`.

Reply:
Done.